main.py

```diff
@@ -2,6 +2,47 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
```

Comment:
There are typos in the actor names 'George Klooney' and 'Kianu Reaves'. These are corrected later in the code, but it's better to initialize them correctly.